petab/v1/priors.py

```diff
@@ -278,7 +278,10 @@ def from_par_dict(
         :return: A distribution object.
         """
         dist_type = d.get(f"{type_}PriorType", C.PARAMETER_SCALE_UNIFORM)
```

Comment:
IMO the default can come first, and the override can then be stated in the "positive" case, instead of `not`s. This is how it's mostly done elsewhere in the code as I recall.

But fine as is too.

This suggestion is for the changed code too, but github wouldn't let me edit the block.

```suggestion
        dist_type = C.PARAMETER_SCALE_UNIFORM
        if _table_dist_type := d.get(f"{type_}PriorType") and (isinstance(_table_dist_type, str) or not np.isnan(_table_dist_type)):
            dist_type = _table_dist_type
```